Python/ceval_macros.h

```diff
@@ -78,7 +78,12 @@
 #   define TAIL_CALL_ARGS frame, stack_pointer, tstate, next_instr, instruction_funcptr_table, oparg
 #endif
 
-#if _Py_TAIL_CALL_INTERP
+#if _Py_TAIL_CALL_INTERP && (defined(__clang__) || defined(__GNUC__))
```

Comment:
```suggestion
#if _Py_TAIL_CALL_INTERP
#if (defined(__clang__) || defined(__GNUC__))
```
Maybe indent the rest as well.